src/gh/issue.rs

```diff
@@ -33,7 +33,7 @@ impl GetDetail for Issue {
     fn get_detail(&self) -> String {
         let title = self.title.to_string();
         format!(
-            "{} {} {}\n{}",
+            "{} [{}] {}\n{}",
```

Comment:
We should make this markdown I think
```suggestion
            "# {} [{}] {}\n\n{}",
```